benchmarks/src/score.rs

```diff
@@ -0,0 +1,206 @@
+use std::time::Duration;
```

Comment:
We needed some more abstract types than integers to be able to support the scoring hierarchy required by the spec, so I introduced a few here. We also needed types to model the JSON format that signal-processing expects as well.